components/commands/common/accelerator_parsing.cc

```diff
@@ -33,6 +35,28 @@ constexpr char kApplicationNew[] = "AppNew";
 constexpr char kMenu[] = "Alt";
 constexpr char kRMenu[] = "AltGr";
 
+struct ModifierName {
+  const ui::KeyEventFlags modifier;
+  const base::StringPiece name;
+};
+
+const std::vector<ModifierName>& GetAllModifierNames() {
+  static const base::NoDestructor<std::vector<ModifierName>> modifier_names({
+#if BUILDFLAG(IS_MAC)
+    {.modifier = ui::EF_COMMAND_DOWN, .name = "Command"},
+#else
+    {.modifier = ui::EF_COMMAND_DOWN, .name = "Meta"},
+#endif
```

Comment:
cc. @fallaciousreasoning , @petemill 

This part seems to have broken the migration, as "Meta" isn't considered as modifier names anymore, "Meta" was dropped at runtime.

Reply:
Oh yeah, that'd do it! On the bright side, it shouldn't happen for anyone who didn't have shortcuts enabled on MacOS when this landed :smile:

Reply:
We add/remove accelerators, so we'd try and remove `ui::Accelerator("B", 0)` which isn't added. Basically, we're getting stuck in a situation where:
1) We have "Meta+B" in our prefs
2) We parse it as "B" (because we don't understand `Meta`)
3) We see it used to be a default, and isn't now, so we try and remove "B" from our prefs
4) There is no "B" accelerator in our prefs (there is a "Meta+B" though), so we don't do anything.

Reply:
Yeah, that makes perfect sense. Let's see what we can do here!